tests/acceptance/testall

```diff
@@ -504,21 +509,23 @@ START_TIME=$(unix_seconds)
   echo -n "" > "$XMLTMP"
 
 for test in $TESTS; do
-  echo "    "\<testcase name=\"$(basename $test)\" >> "$XMLTMP"
+  echo "    "\<testcase name=\"`basename $test`\" >> "$XMLTMP"
   echo "              "classname=\"$test\" >> "$XMLTMP"
+  set +e # don't fail on errors when running the tests - this is expected
```

Comment:
Why not adding a || true on commands that can fail.
Then we still can disable set -e globally